docs/pages/color_usage.js

```diff
@@ -0,0 +1,270 @@
+        `}
+      />
+      <MainSection
+        name="Primary color"
+        description={`
+        A primary color is the color displayed most frequently across screens and components, used to convey high-emphasis actions. Our primary color related tokens are:
+        **$color-background-primary-base** - Use this token when conveying a primary action.
+        **$color-background-brand** - Use this token when a background color is needed to signify the Pinterest brand.        `}
+      >
+        <ColorTile
+          description="Primary base"
+          textColor="white"
+          fullTokenName="color-background-primary-base"
+        />
+      </MainSection>
+      <MainSection
+        name="Secondary color"
+        description={`
+        A secondary color highlights medium to low-emphasis actions, creating a balance with the primary color. Our secondary color related token is:
+        **$color-background-secondary-base** - Use this token for secondary, medium to low-emphasis actions.
+        `}
+      >
+        <ColorTile description="Secondary base" fullTokenName="color-background-secondary-base" />
+      </MainSection>
+      <MainSection
+        name="Accent gray color"
+        description={`
+        Our tertiary color offers ways to accent the UI when the primary color doesn't work well on the proposed layout. Our accent gray related token is:
+        **$color-background-tertiary-base** - Use this token on medium to low-emphasis actions.
+        `}
+      >
+        <ColorTile
+          description="Tertiary base"
+          textColor="white"
+          fullTokenName="color-background-tertiary-base"
+        />
+      </MainSection>
+      <MainSection
+        name="Shopping color"
+        description={`
+        Instead of the Pushpin 450, the Skycicle 500 (blue) is the primary color to indicate shopping-related features. The related token is:
+        **$color-background-shopping** - Use this token name when conveying a primary action on shopping experiences.
+        `}
+      >
+        <ColorTile
+          description="Shopping"
+          fullTokenName="color-background-shopping"
+          name="Skycicle"
+          number={500}
+        />
+      </MainSection>
+      <MainSection
+        name="Background color"
+        description="Background colors are used behind text or applied to other elements to deliver specific messages or meanings, such as information, success, warning, and error. They are divided into two categories, base colors and weak colors. "
+      >
+        <MainSection.Subsection
+          title="Base background colors"
+          description={`
+          Use on UI elements to convey a specific status or message. Usually, they aren't used behind default text.
+          **$color-background-info-base** - Use this token as a background color to indicate neutral information.
```

Comment:
"Use the token(s)", "Use this token", etc are a bit repetitive in this section. I wonder if they're needed?